Doc/reference/expressions.rst

```diff
@@ -1367,7 +1367,7 @@ This operation can be customized using the special :meth:`~object.__add__` and
 The ``-`` (subtraction) operator yields the difference of its arguments.  The
 numeric arguments are first converted to a common type.
 
-This operation can be customized using the special :meth:`~object.__sub__` method.
+This operation can be customized using the special :meth:`~object.__sub__` and :meth:`~object.__rsub__` methods.
```

Comment:
For convenience, here's the suggested wrapping in the form of an actual suggestion:

```suggestion
This operation can be customized using the special :meth:`~object.__sub__`
and :meth:`~object.__rsub__` methods.
```